docs/articles_en/openvino_workflow/running_inference_with_openvino/dldt_deployment_optimization_guide/preprocessing_overview/torchvision_preprocessing_converter.rst

```diff
@@ -0,0 +1,69 @@
+.. {#torchvision_preprocessing_converter}
+
+Torchvision preprocessing converter
+=======================================
+
+
+.. meta::
+   :description: See how OpenVINO™ enables torchvision preprocessing to optimize model inference.
+
+
+Images used as input for AI models often need to be preprocessed, to have proper dimensions or data type.
+For a torchvision preprocessing pipeline, you can use the **torchvision.transforms** OpenVINO feature, instead of 
+a separate library that would add an additional pipeline step.
+
+The Torchvision-to-OpenVINO converter automatically translates operators to the OpenVINO format and embeds them 
+in your OpenVINO model, reducing the overall application complexity and enabling additional performance optimizations.
+
+
+.. note:: 
+
+   Not all torchvision transforms are supported yet. The following operations are available: 
+
+   .. code-block::
+
+      transforms.Compose
+      transforms.Normalize
+      transforms.ConvertImageDtype
+      transforms.Grayscale
+      transforms.Pad
+      transforms.ToTensor
+      transforms.CenterCrop
+      transforms.Resize
+
+
+Example
+###################
+
+.. code-block:: py
+
+   preprocess_pipeline = torchvision.transforms.Compose(
```

Comment:
@p-wysocki we usually keep code snippets in python files https://github.com/openvinotoolkit/openvino/tree/master/docs/snippets

Reply:
That's simplified code, the one I gave in the PR description when adding torchvision preprocessor. It will require some work to be actually runable (for example, the function for SimpleConvnet needs to be copied from tests). 

I suggest merging it as is for now and continuing this topic in the ticket 129284.